docs/userguides/quickstart.md

```diff
@@ -96,6 +96,45 @@ sudo rm -rf **\~/.solcx**
 sudo rm -rf **\~/.vvm**
 ```
 
+## Overview
+
+The ApeWorx project folder hierarchy:
+
+```
+project              # The root project directory
+├── contracts/       # Project source files, such as '.sol' or '.vy' files
+├── tests/           # Project tests, ran using the 'ape test' command
+├── scripts/         # Project scripts, such as deploy scripts, ran using the 'ape run <name>' command
+└── ape-config.yaml  # The ape project configuration file
+```
+
+## Enviroment Variables:
+
+Enviroment Variables are used to help connect you to your files or ecosytems outside of ApeWorx
+
+Please setup environment variables (where applicable) and follow the latest instructions from the 3rd party:
+
+Example of a use case:
```

Comment:
```suggestion
Example use cases:
```

Reply:
Also, maybe link to one of the plugin's READMEs

Reply:
are you talking about docs/userguides/plugins.md?

and how does that help?

Reply:
issue created and will be fixed https://github.com/ApeWorX/ape-infura/issues/28